css/colors.css

```diff
@@ -211,7 +211,7 @@ table {
 caption {
```

Comment:
what is this file?

Reply:
this file is for the color.html page. There are some styling that is specific for this page, and I don't want to include it in the main.css, so I create another css for it.